### ISSUE TITLE
Recent versions of python notebook package have API changes 

### ISSUE THREAD

**Issue by asymingt** [michael-quinlan/jupyter-bazel#4]
I found that in more recent versions of `notebook` you'll need to change this:

```
from notebook.notebookapp import main
```

... to this ...

```
from notebook.app import main
```

... or else you will get a ModuleImportError.

**Comment by michael-quinlan** [comment]
Yes, I found some changes like this and fixed in my main (mono) repo. I'll back port some changes to this public example code. 

Thanks!
